lib/core/viewmodels/not_found_viewmodel.dart

```diff
@@ -0,0 +1,20 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:stacked/stacked.dart';
+
+// SERVICE
+import 'package:notredame/core/services/navigation_service.dart';
+
+// OTHER
+import 'package:notredame/locator.dart';
+import 'package:notredame/core/constants/router_paths.dart';
+
+class NotFoundViewModel extends BaseViewModel {
+  /// Used to redirect on the dashboard.
+  final NavigationService _navigationService = locator<NavigationService>();
+
+  NotFoundViewModel();
```

Comment:
Because we aren't supposed to access this page, I think that should be very interesting to log the event. Could you use `AnalyticsService` (`logEvent` function) to report this information, please? 😄